tasks.py

```diff
@@ -140,25 +140,39 @@ def type_coverage(ctx):
         "packages": f"One or more packages to type-check with mypy. (Default: {DEFAULT_PACKAGES_TO_TYPE_CHECK})",
         "show-default-packages": "Print the default packages to type-check and then exit.",
         "install-types": "Automatically install any needed types from `typeshed`.",
+        "daemon": "Run mypy in daemon mode with faster analysis."
+        " The daemon will be started and re-used for subsequent calls."
+        " For detailed usage see `dmypy --help`.",
```

Comment:
<img width="759" alt="mypy_times" src="https://user-images.githubusercontent.com/13108583/182169290-e69660f9-30f8-4c78-8397-17d541688681.png">

☝️ Timing `mypy` runs.

1. First `mypy` run with a clean cache.
2. Second `mypy` run with a loaded cache. Slight improvement (~1 second).
3. First `dmypy` run without daemon running. About the same time as normal `mypy` check.
4. Second `dmypy` run with daemon already running. Massive improvement. `~7.5 seconds` to less than `0.2 seconds`

Reply:
cool! If the cache get's in a bad state how does one clear it?

Reply:
You would delete the `.mypy_cache/` directory that sits at the project root.

I can add a convenience `--clear-cache` flag to `invoke type-check` to make this easier.